app/config/collections.php

```diff
@@ -963,12 +963,12 @@
             ],
             [
                 '$id' => 'expire',
-                'type' => Database::VAR_INTEGER,
+                'type' => Database::VAR_DATETIME,
                 'format' => '',
                 'size' => 0,
                 'signed' => false,
                 'required' => false,
-                'default' => 0,
+                'default' => null,
                 'array' => false,
                 'filters' => [],
```

Comment:
```suggestion
                'filters' => ['dateFormat'],
```

Reply:
seems filters are missing everywhere for datetime

Reply:
We need to merge this PR in 
[3688](https://github.com/appwrite/appwrite/pull/3688)

Reply:
@fogelito, perhaps you can cherry-pick those commits into this branch to prevent merging in changes from the [refactor-permissions-inc-console-fix](https://github.com/appwrite/appwrite/tree/refactor-permissions-inc-console-fix) branch?